Tests/LibJS/test-bytecode-js.cpp

```diff
@@ -111,15 +110,13 @@ TEST_CASE(loading_multiple_files)
     }
 
     {
-        auto test_file_script_or_error = JS::Script::parse("if (f() !== 'hello') throw new Exception('failed'); "sv, ast_interpreter->realm());
-        EXPECT(!test_file_script_or_error.is_error());
+        auto test_file_script = MUST(JS::Script::parse(
+            "if (f() !== 'hello') throw new Exception('failed'); "sv, ast_interpreter->realm()));
 
-        auto test_file_script = test_file_script_or_error.release_value();
         auto const& test_file_program = test_file_script->parse_node();
 
         auto executable = MUST(JS::Bytecode::Generator::generate(test_file_program));
-        auto result = bytecode_interpreter.run(*executable);
-        EXPECT(!result.is_error());
+        MUST(bytecode_interpreter.run(*executable));
```

Comment:
to confirm, this file has MUST because we cannot format() a `JS::Script` (or w/e the type is here) ? or are there some missed opportunities for TRY_OR_FAIL

Reply:
Yes, it's due to a missing `Formatter<>` implementation.

This is due to `JS::Completion` and not `JS::Script` (afterall, we try to print the error-value, not the success-value), but yes, that's it. Implementing that Formatter feels like it's outside the scope of this PR.

I'll add a TODO, because you're right, failing would be nicer than crashing.